geode-core/src/main/java/org/apache/geode/net/SSLParameterExtension.java

```diff
@@ -29,9 +29,9 @@
   /**
    * Initialize the SSLParameterExtension.
    *
-   * @param config the DistributionConfig
+   * @param properties the Properties
    */
-  default void init(DistributionConfig config) {}
+  default void init(Properties properties) {}
```

Comment:
this is a breaking change on a public class that is not marked @Experimental.

why not keep `init(DistributionConfig)` and just make it call `init(distributionConfig.toProperties())` ?

Reply:
The API break is necessary because the original was not a valid API since it leaked internal types. The case of two wrongs make a right.